packaging/calculix-precice2_2.17-1_amd64/usr/share/doc/calculix-precice2/copyright

```diff
@@ -0,0 +1,13 @@
+Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: preCICE
+Upstream-Contact: preCICE mailing-list <info@precice.org>
+Copyright: (c) 2019 the preCICE developers <info@precice.org>
```

Comment:
Question (I have no idea): Why 2019? Are we aiming for the oldest? The newest? A range? Is there a guideline?

Reply:
I copy pasted from the `libprecice` packages, so, no idea. I'm curious too. Guido Dhondt still uses a date from the last century, so probably not "newest".

Reply:
I understand that the year should be the date of publication (of what, I will touch in a moment). If we keep publishing, we should also make this a range of years. This is what I understand from this page: https://www.copyrightlaws.com/copyright-symbol-notice-year/

Now, the 2019 is probably the year we first had Debian packages for preCICE. If a package and the corresponding source are two different things is a different story. I would tend to write "2021" here, pointing to the package date. Or we add something like 2016-2021, covering the first state of the adapter, till today.

@uekerman do you maybe know more about this topic?

Reply:
@boris-martin You copy-pasted the output instead of the input :wink: 

https://github.com/precice/precice/blob/23ef51b8f429956ea620d8a9164e513bb074cd3a/tools/releasing/bumpversion.sh#L58-L64

Reply:
Ah yes, didn't spot this automation, I could add it here.
Which means we'll end up with 2021 then ?
(I wonder how I got 2019 then, that's some old output)

Reply:
You may :wink:

Reply:
Actually I'm a bit confused (and very bad at reading Bash),  but this is more for the control file than copyright no?
(In the meantime I just updated to 2021 in the copyright)

I feel moderately comfortable adding this while we don't have a consistent versioning scheme.

Reply:
I think this is the information that package managers will show as license.
By the way, I notice there is a `Files: *` next to each license. Can we maybe use this to distinguish between CalculiX files and own files? Some files will have shared copyright, some will be distinct preCICE copyright.

Reply:
That's the theory, but here the package only consists of the compiled executable, so it's not really useful. But I'd do this if I packaged the source so we can give single copyright to our helper files etc.

Reply:
Then everything is fine and I think we can resolve this discussion.